client/src/components/Routers.js

```diff
@@ -31,23 +33,26 @@ export default function Routers() {
                 <Login />
               </Route>
               { appContext.routes
-                .map(({linkTo, exact = false, component}, idx) => (
+                .map(({linkTo, exact = false, component, disabled}, idx) => (
                   <PrivateRoute
                     path={linkTo}
-                    component={component}
+                    component={disabled ? Unauthorized : component}
                     exact={exact}
                     getScrollContainerRef={linkTo === '/verify' ? () => refContainer.current : undefined}
                     key={`route_${idx}`}
                     />
                 ))
               }
               <Route
-                render={({location}) => (
-                  <Redirect to={{
-                    pathname: appContext.user ? "/" : "/login",
-                    state: { from: location }
-                  }}/>
-                )}
+                render={
+                  appContext.user ? (Page404) :
```

Comment:
Why haven't undefined user cause 404 page rather than unauthorized page?

Reply:
Undefined user (i.e. not logged in) should always redirect to `/login`.
The unauthorized page is reserved for logged in users trying to access something they don't have permission to see.